src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Entities/Enums/RelationshipType.cs

```diff
@@ -0,0 +1,309 @@
+    /// </summary>
+    DELEGATED_TO,
+
+    /// <summary>
+    /// The from Element depends on each to Element, during a LifecycleScopeType period.
+    /// </summary>
+    DEPENDS_ON,
+
+    /// <summary>
+    /// The from Element is a descendant of each to Element.
+    /// </summary>
+    DESCENDANT_OF,
+
+    /// <summary>
+    /// The from Element describes each to Element. To denote the root(s) of a tree of elements in a collection, the rootElement property should be used.
+    /// </summary>
+    DESCRIBES,
+
+    /// <summary>
+    /// The from Vulnerability has no impact on each to Element. The use of the doesNotAffect is constrained to VexNotAffectedVulnAssessmentRelationship classed relationships.
+    /// </summary>
+    DOES_NOT_AFFECT,
+
+    /// <summary>
+    /// The from archive expands out as an artifact described by each to Element.
+    /// </summary>
+    EXPANDS_TO,
+
+    /// <summary>
+    /// The from Vulnerability has had an exploit created against it by each to Agent.
+    /// </summary>
+    EXPLOIT_CREATED_BY,
+
+    /// <summary>
+    /// Designates a from Vulnerability has been fixed by the to Agent(s).
+    /// </summary>
+    FIXED_BY,
+
+    /// <summary>
+    /// A from Vulnerability has been fixed in each to Element. The use of the fixedIn type is constrained to VexFixedVulnAssessmentRelationship classed relationships.
+    /// </summary>
+    FIXED_IN,
+
+    /// <summary>
+    /// Designates a from Vulnerability was originally discovered by the to Agent(s).
+    /// </summary>
+    FOUND_BY,
+
+    /// <summary>
+    /// The from Element generates each to Element.
+    /// </summary>
+    GENERATES,
+
+    /// <summary>
+    /// Every to Element is a file added to the from Element (from hasAddedFile to).
+    /// </summary>
+    HAS_ADDED_FILE,
+
+    /// <summary>
+    /// Relates a from Vulnerability and each to Element with a security assessment. To be used with VulnAssessmentRelationship types.
+    /// </summary>
+    HAS_ASSESSMENT_FOR,
+
+    /// <summary>
+    /// Used to associate a from Artifact with each to Vulnerability.
+    /// </summary>
+    HAS_ASSOCIATED_VULNERABILITY,
+
+    /// <summary>
+    /// The from SoftwareArtifact is concluded by the SPDX data creator to be governed by each to license.
+    /// </summary>
+    HAS_CONCLUDED_LICENSE,
+
+    /// <summary>
+    /// The from Element treats each to Element as a data file. A data file is an artifact that stores data required or optional for the from Element's functionality. A data file can be a database file, an index file, a log file, an AI model file, a calibration data file, a temporary file, a backup file, and more. For AI training dataset, test dataset, test artifact, configuration data, build input data, and build output data, please consider using the more specific relationship types: trainedOn, testedOn, hasTest, configures, hasInput, and hasOutput, respectively. This relationship does not imply dependency.
```

Comment:
Can we split this comment into multiple lines to make it easier to read without have to scroll multiple screens to the right?